crates/nu-cli/src/syntax_highlight.rs

```diff
@@ -429,15 +367,14 @@ fn find_matching_block_end_in_expr(
                     None
                 }
             }
+
+            // all other non-nested expressions
+            _ => None,
         };
     }
     None
 }
 
-fn get_char_at_index(s: &str, index: usize) -> Option<char> {
-    s[index..].chars().next()
-}
-
-fn get_char_length(c: char) -> usize {
-    c.to_string().len()
+fn slice(line: &str, start: usize, end: usize, offset: usize) -> &str {
```

Comment:
Maybe worth expanding this fn name so you don't have to look it up.

Reply:
Yep, and will add documentation.